extensions/amp-iframe/1.0/component.js

```diff
@@ -89,22 +90,28 @@ export function Iframe({
     if (!win) {
       return;
     }
-    const io = new win.IntersectionObserver((entries) => {
-      const last = entries[entries.length - 1];
-      isIntersectingRef.current = last.isIntersecting;
-      if (last.isIntersecting || !dataRef.current || !win) {
-        return;
-      }
-      attemptResize();
-    });
-    io.observe(iframe);
+
     win.addEventListener('message', handlePostMessage);
 
     return () => {
-      io.unobserve(iframe);
       win.removeEventListener('message', handlePostMessage);
     };
-  }, [attemptResize, handlePostMessage]);
+  }, [handlePostMessage]);
+
+  const ioCallback = useCallback(
+    ({isIntersecting}) => {
+      if (isIntersecting === isIntersectingRef.current) {
+        return;
+      }
+      isIntersectingRef.current = isIntersecting;
+      if (!isIntersecting) {
```

Comment:
We should probably check if there's a pending resize to avoid the `console.error` call.

Reply:
This still isn't checking for a pending resize.

Reply:
Done.